mono/unit-tests/test-path.c

```diff
@@ -26,12 +26,12 @@ make_path (const char *a, int itrail, int slash, int upcase)
 int
 main (void)
 {
-	static const char * const bases [2] = {"/", "/foo"};
-	static const char * const files [3] = {"/foo", "/foo/bar", "/foob"};
+	static const char * const bases [2] = {"/", "/1"};
+	static const char * const files [6] = {"/1", "/1/2", "/1/2/3", "/12", "/2", "/2/2/"};
```

Comment:
but now you lost the case sensitive tests on win32...

Reply:
Oops good point. I'll go with a/b/c and such (and change the upcasing)